lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -545,13 +545,7 @@ states:
       next:
         targetState: ADDRESS_AND_FRAUD_J2
       access-denied:
-        targetState: MULTIPLE_DOC_F2F_CHECK_PAGE
-        checkIfDisabled:
-          f2f:
-            targetState: MULTIPLE_DOC_CHECK_PAGE
```

Comment:
Are you sure we want to remove this check? I think we probably need to keep this, as we can't start routing users to F2F if it's disabled.

Reply:
Ah you're right, PROVE_ANOTHER_WAY_J2 has a f2f route